stories/create-stories-from-doc.js

```diff
@@ -1,8 +1,11 @@
 'use strict'
 
 const fs = require('node:fs')
+const path = require('node:path')
 const puppeteer = require('puppeteer') // eslint-disable-line import/no-extraneous-dependencies
 
+const version = '5.3'
```

Comment:
Note for myself: need to check if the 5.3 strings in this PR are well modified when executing `npm run release-version 5.3 5.4`

Reply:
Nevermind, it's `npm run release-version 5.3.0 5.4.0` so it must be updated manually like other versions in the doc at this moment.